resources/views/livewire/password-strength.blade.php

```diff
@@ -0,0 +1,35 @@
+<div>
```

Comment:
```suggestion
<div style="display: contents; background-color: inherit;">
```
Needed to make the form elements work with form classes